app/screens/channel_members/channel_members.js

```diff
@@ -319,15 +296,22 @@ class ChannelMembers extends PureComponent {
                     style={{marginVertical: 5}}
                 >
                     <SearchBar
-                        ref={this.searchBarRef}
-                        placeholder={formatMessage({id: 'search_bar.search', defaultMesage: 'Search'})}
-                        height={27}
-                        fontSize={14}
-                        textColor={changeOpacity('#000', 0.5)}
-                        hideBackground={true}
-                        textFieldBackgroundColor={'#fff'}
-                        onChange={this.searchProfiles}
-                        onSearchButtonPress={this.onSearchButtonPress}
+                        ref='search_bar'
+                        placeholder={formatMessage({id: 'search_bar.search', defaultMessage: 'Search'})}
+                        cancelTitle={formatMessage({id: 'mobile.post.cancel', defaultMessage: 'Cancel'})}
+                        backgroundColor='transparent'
+                        inputHeight={30}
+                        inputStyle={{
```

Comment:
Same here. Perhaps if we're defining these same styles repeatedly, we should also make this into a component so we don't need to change it in 5 different places if we decide we want the search bar to look different